src/lib/Microsoft.Health.Fhir.Ingest/Service/NormalizationEventConsumerService.cs

```diff
@@ -158,7 +203,9 @@ private void NormalizeMessage(JObject token, IEventMessage evt, IContentTemplate
                 catch (Exception ex)
                 {
                     // Translate all Normalization Mapping exceptions into a common type for easy identification.
-                    throw new NormalizationDataMappingException(ex);
+                    var mappingException = new NormalizationDataMappingException(ex);
+                    mappingException.AddEventContext(evt);

```

Comment:
Suggestion: You can make AddEventContext return the exception allowing you to chain statements fluent style, i.e. 
```
throw new NormalizationDataMappingException(ex)
    .AddEventContext(evt);
```